src/users/users.test.ts

```diff
@@ -53,4 +56,10 @@ describe('Users', () => {
     resultSingle = users.readById(1);
     await resultSingle.then((result) => expect(result).toStrictEqual({}));
   });
+
+  test('update', async () => {
+    mock.onPut(updatePath).reply(200, { Success: true, Results: [{}] });
+    resultUpdateRaw = users.update({ id: 1 });
+    await resultUpdateRaw.then((result) => expect(result).toMatchObject({ data: {} }));
```

Comment:
I see that it's the same in the test above but I wouldn't do this, there is no need for `then` when there is `await`. So `expect` can be outside of the arrow function:
```suggestion
    expect(await resultUpdateRaw).toMatchObject({ data: {} });
```